uberfire-extensions/uberfire-metadata/uberfire-metadata-backends/uberfire-metadata-backend-elasticsearch/pom.xml

```diff
@@ -114,6 +124,14 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-to-slf4j</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+    </dependency>
```

Comment:
what happens if we just exclude the dependencies? I think we use anyway slf4j or other bridge providers than log4j

Reply:
I agree, removing them...

Reply:
removed, relaunching FDB prod on https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1461